deployment/ansible/roles/model-my-watershed.celery-worker/templates/nginx-flower.conf.j2

```diff
@@ -0,0 +1,37 @@
+server {
+  listen *:80;
+  server_name _;
+
+  access_log /var/log/nginx/flower.access.log logstash_json;
+
+  {% if ['packer'] | is_in(group_names) -%}
+  location /version.txt {
+    alias /srv/version.txt;
```

Comment:
This appears to have been an inconsistency in the SHA produced by `git describe`, which gets used when `GIT_COMMIT` is not supplied. Should be resolved in https://github.com/WikiWatershed/model-my-watershed/commit/f66a4f300d41ac4c8cfc394b430aed168c350461.